### DISCUSSION TITLE
Multi-screen support

### DISCUSSION THREAD

**Discussion by MichelV69** [LocuraDU/DU-Locura-Storage-Info#2] [Q&A] [answered]
I notice in the code that you actively scan for multiple screens.  However, it doesn't look like anything "happens" with the second screen.  Am I missing something?

**Comment by CredenceHamby** [comment, marked as answer]
I have started adding support for this, just haven't completed it, but it's on the way.

**Comment by CredenceHamby** [comment]
Multi screen/hub support in next version.

v2.2 (BETA) & v2.3 (ALPHA) (MERGE)
[ADDED] Option to reverse the RGB colors for the percent bar.
[ADDED] Option to show how many items left in container.
[ADDED] Up to 5 screens, and 5 container/hub support.
[ADDED] Automatically turns on screens if screens are off.
[CHANGED] Header bar item name is now all caps.

https://github.com/LocuraDU/DU-Locura-Storage-Info/wiki/Changelog
